### ISSUE TITLE
Display issue with Switch on iOS

### ISSUE THREAD

**Issue by tjvantoll** [NativeScript/theme#174]
I think I’ve found a small display issue with `checked="false"` Switch components on iOS. To recreate you can use this markup.

``` HTML
<ActionBar title="My App"></ActionBar>

<GridLayout class="page">
    <StackLayout>
        <Switch checked="true"></Switch>
        <Switch checked="false"></Switch>

        <!-- A workaround I’m using for now that seems to work -->
        <Switch checked="false" offBackgroundColor="#FAFAFA"></Switch>
    </StackLayout>
</GridLayout>
```

Here’s what that looks like on an iPhone XS Simulator. To me the display of the `checked="false"` switch looks off.

![Screen Shot 2019-08-29 at 12 05 02 PM](https://user-images.githubusercontent.com/544280/63956905-60687480-ca55-11e9-95e7-7b5f69ba085a.png)

The potential problem seems isolated to iOS as the display on Android seems fine.

![Screen Shot 2019-08-29 at 12 11 26 PM](https://user-images.githubusercontent.com/544280/63957308-23e94880-ca56-11e9-9a74-bc2304f81937.png)

**Comment by bundyo** [comment]
Yeah, looks and acts strange - a bit jumpy. Does the workarounded one work okay? Anyway, I'll look at it.

**Comment by bundyo** [comment]
Should be fixed in 2.0.10.
